src/contexts/DanteFinance/DanteFinanceProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { createContext, useEffect, useState } from 'react';
+import { useWallet } from 'use-wallet';
+
+import config from '../../config';
+import DanteFinance from '../../dante-finance';
+
+export interface DanteFinanceContext {
```

Comment:
This is not the Context this is the data inside the context. So it should be named `DanteFinanceContextData`